tests/tensorflow/data/reference_graphs/2.11/pruning/filter_pruning/inception_v3.dot

```diff
@@ -0,0 +1,3468 @@
+strict digraph  {
```

Comment:
There are some changes to inception model graphs. There are two types of change:
- "inception_v3/batch_normalization_\*/**scale**" [op=Const]; -> "inception_v3/batch_normalization_\*/**Const**" [op=Const];
- "inception_v3/batch_normalization_\*/**scale**" [op=**Placeholder**]; -> "inception_v3/batch_normalization_\*/**Const**" [op=**Const**];

 Feel free to download the old and the new versions and see the diff.